src/olympia/scanners/tests/test_actions.py

```diff
@@ -392,6 +392,45 @@ def test_delay_auto_approval_indefinitely_and_restrict(self):
                 f'Addon {addon.pk} Version {addon.current_version.pk}.'
             )
 
+    def test_delay_auto_approval_indefinitely_and_restrict_with_ipv6(self):
```

Comment:
nit, but this is a copy and paste of the previous test while not _just_ testing ipv6 - only one of the users has a ipv6 last_login_ip.  I don't have an easy suggestion of what to change but my "feeling" is the tests need restructuring/de-duplicating/refactoring.

Reply:
Yeah, I also felt this wasn't ideal. I tried to remove extra cruft and focus only on the network restriction part while still keeping some non-ipv6 data to prove it didn't impact anything else. I could have re-used on the existing tests instead of creating a new one but I felt it was going to make it more confusing... Ultimately this does demonstrate the problem & fix